TCAT.xcodeproj/project.pbxproj

```diff
@@ -525,7 +530,7 @@
 				TargetAttributes = {
 					449A7C741D80D0E80019300C = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = ZGMCXU7X3U;
+						DevelopmentTeam = TD73A242XE;
```

Comment:
Keep this to the one we had it. It might be fine, but change it back just in case